### PR TITLE
fix: sidebar hiding some elements

### PR DESCRIPTION
**Pull request by thr0wn**
<table>
  <tr>
    <td>Issue</td>
     <td>Fix</td>
  </tr>
  <tr>
    <td></td>
    <td></td>
  </tr>
 </table>

No side effects found on other screens:
<table>
  <tr>
    <td></td>
<td></td>
<td></td>
<td></td>
  </tr>
 </table>

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -15,47 +15,47 @@
       "resolved": "https://registry.npmjs.org/@babel/code-frame/-/code-frame-7.8.3.tgz",
       "integrity": "sha512-a9gxpmdXtZEInkCSHUJDLHZVBgb1QS0jhss4cPP93EW7s+uC5bikET2twEF3KV+7rDblJcmNvTR7VJejqd2C2g==",
       "requires": {
-        "@babel/highlight": "7.8.3"
+        "@babel/highlight": "^7.8.3"
```
Comment (pedroferreira1) [inline package-lock.json:18]: I don't think we should change anything on the `package-lock.json`. There are many version changes that could impact the wallet.

**Review by pedroferreira1 — CHANGES_REQUESTED** [review]

**Comment by pedroferreira1** [comment]
Check this here. I think it was because of the nav position change from `fixed` to `sticky`. Is there a reason to change?

bug-nav

File: package-lock.json (hunk 1 above)
Comment (thr0wn) [reply package-lock.json]: You are right. All changes on `package-lock.json` were reverted, thanks.

**Comment by thr0wn** [comment]
The purpose of position `sticky` was to consider the nav `height` while rendering the page, but I reverted as it caused some troubles with electron. Seems to be working fine now.

**Review by pedroferreira1 — APPROVED** [review]

**Review by obiyankenobi — APPROVED** [review]

**Pull request merged**